appIcons.js

```diff
@@ -426,6 +446,15 @@ function activateAllWindows(app, settings) {
     }
 }
 
+/**
+ * This closes all windows of the app.
+ */
+function closeAllWindows(app, settings) {
+    let windows = getInterestingWindows(app, settings);
+    for (let i = 0; i < windows.length; i++)
+        windows[i].delete(global.get_current_time());
+}
+
```

Comment:
Great idea to simplify the code. Implemented here: [9cb1c49b91ce08cb509e1da9d8497fe5faf69f34]